extensions/objects/templates/svg-filename-editor.component.ts

```diff
@@ -405,25 +416,30 @@ angular.module('oppia').component('svgFilenameEditor', {
       };
 
       var loadGroupedObject = function(objId, obj, groupedObjects) {
-        // Checks the first five characters of the id.
-        if (objId.slice(0, 5) === 'group') {
-          if (groupedObjects.length <= objId.slice(5)) {
+        // Checks the first five characters of the id to identify the
+        // svg objects that are grouped together.
```

Comment:
May be better to say "Checks if the id starts with 'group' to indicate.."?

Reply:
Done.